src/commands/dev.ts

```diff
@@ -148,6 +149,17 @@ export async function command(commandOptions: CommandOptions) {
     }
   }
 
+  function addToDependents(spec: string, fileLoc: string) {
+    if (!spec.includes('web_modules') && !spec.includes('node_modules') && !fileLoc.includes('web_modules') && !fileLoc.includes('node_modules')) {
```

Comment:
one of the nice things about web_modules is that there really aren't many files inside of it, and they're all valid web-ready ESM, so IMO it's fine to not make special rules for them, even if we don't expect many edits to the web_modules directory.